README.md

```diff
@@ -768,6 +769,17 @@ $ cmake .. -G "Unix Makefiles"
 $ make
 ```
 
+#### 5.1.1 CentOS/RHEL 7
+
+CentOS/RHEL 7 comes with gcc 4.8, which is compatible with C++11, but unfortanely ships a broken implementation of `<regex>`.
+We work around this issue by falling back to using `boost::regex` in this case.
+
+To build exiv2 on CentOS, please install the following additional packages:
+- `cmake3`
```

Comment:
I do not think the cmake package should be listed here. It is already a dependency for building the project, even if boost is not going to be used.

Reply:
I added it for completeness, as the `cmake` package is cmake 2.8, but we require cmake 3.